hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -104,6 +105,19 @@ public class HoodieAvroUtils {
 
   public static final Schema RECORD_KEY_SCHEMA = initRecordKeySchema();
 
+  /**
+   * TODO
```

Comment:
what's the plan here?

Reply:
This todo left by Alexey. I guess he want to support other type of records serialization. My plan is that record will be converted to avroRecord before pass in dataBlock.